plaid/plaid.go

```diff
@@ -29,16 +29,17 @@ type Client struct {
 }
 
 type ClientOptions struct {
-	ClientID    string
-	Secret      string
-	PublicKey   string
-	Environment Environment
-	HTTPClient  *http.Client
+	ClientID                  string
+	Secret                    string
+	PublicKey                 string
+	Environment               Environment
+	HTTPClient                *http.Client
+	SkipEnvironmentValidation bool
```

Comment:
So we need some way of being able to initialize the client with an internal api address, this would allow us to do so - pretty hacky, WDYT?

Reply:
alternatively, we can remove the validation all together

Reply:
I'd be in favor of removing the validation instead of passing in this flag, perhaps we can print a warning instead of erroring if clients don't pass in an expected value - we could tell them to `please pass in plaid.Sandbox, plaid.Development, or plaid.Production`

Reply:
⚡️ nice idea! going with that